spatialconnect/src/main/java/com/boundlessgeo/spatialconnect/services/SCBackendService.java

```diff
@@ -532,6 +542,102 @@ private String getAndroidVersion() {
+                .flatMap(new Func1<SCDataStore, Observable<SCSpatialFeature>>() {
+                    @Override
+                    public Observable<SCSpatialFeature> call(SCDataStore scDataStore) {
+                        return scDataStore.storeEdited;
+                    }
+                });
+        //syncs a single store when edits are found
+        syncStores = Observable.combineLatest(syncableStores, connectedToBroker, new Func2<SCSpatialFeature, Boolean, Boolean>() {
+            @Override
+            public Boolean call(final SCSpatialFeature feature, Boolean connected) {
+                Log.d(LOG_TAG, "should sync single store");
+                final ISyncableStore store = (ISyncableStore)dataService.getStoreByIdentifier(feature.getStoreId());
+                syncStore(store);
+                return connected;
+            }
+        });
+
+        connectedToBroker.mergeWith(syncStores)
+                .subscribeOn(Schedulers.newThread())
+                .subscribe();
+    }
+
+    private void syncStores() {
+
+        dataService.getISyncableStores()
+                .subscribe(new Action1<SCDataStore>() {
+                    @Override
+                    public void call(SCDataStore scDataStore) {
+                        syncStore((ISyncableStore) scDataStore);
+                    }
+                });
+    }
+
+    private void syncStore(final ISyncableStore store) {
+        store.unSent().subscribe(new Action1<SCSpatialFeature>() {
+            @Override
+            public void call(final SCSpatialFeature scSpatialFeature) {
+                sensorService.isConnected()
+                        .filter(new Func1<Boolean, Boolean>() {
+                            @Override
+                            public Boolean call(Boolean connected) {
+                                return connected;
+                            }
+                        })
+                        .subscribe(new Action1<Boolean>() {
+                            @Override
+                            public void call(Boolean connected) {
+                                send(scSpatialFeature);
+                            }
+                        });
+            }
+        });
+    }
+
+    private void send(final SCSpatialFeature feature) {
+        final ISyncableStore store = (ISyncableStore)dataService.getStoreByIdentifier(feature.getStoreId());
+        Map<String, Object> featurePayload = store.generateSendPayload(feature);
+        String payload;
+        try {
+            payload = SCObjectMapper.getMapper().writeValueAsString(featurePayload);
+            SCMessageOuterClass.SCMessage message = SCMessageOuterClass.SCMessage.newBuilder()
+                    .setAction(SCCommand.DATASERVICE_CREATEFEATURE.value())
+                    .setPayload(payload)
+                    .build();
+
+//            publishReplyTo(store.syncChannel(), message)
```

Comment:
Feature is not getting sent. publish for now, publishReplyTo can happen when the layer service is integrated.